docs/tools/cmake_in_clickhouse_generator.py

```diff
@@ -1,181 +0,0 @@
-import re
-import os
-from typing import TextIO, List, Tuple, Optional, Dict
```

Comment:
@Felixoid what is the reason for removing this generator? (added in #14711, cc @myrrc )

Reply:
The whole system is deprecated and moved to https://github.com/ClickHouse/clickhouse-docs. It's the only reason, basically.